server/resolvers/TestPlanReport/runnableTestsLengthResolver.js

```diff
@@ -0,0 +1,9 @@
+const runnableTestsLengthResolver = async testPlanReport => {
+    // Custom column added with sequelize.literal provides an array as such
+    // runnableTestsCount: [ x, y, z] where x=jawsCount, y=nvdaCount, z=voCount
+    return testPlanReport.testPlanVersion.dataValues.runnableTestsCount[
+        testPlanReport.atId - 1
+    ];
```

Comment:
Will this work with `.dataValues` removed? Like: `testPlanReport.testPlanVersion.runnableTestsCount[testPlanReport.atId - 1]`.

Reply:
It doesn't seem to work whenever it's through an association like `testPlanVersion` is here. Appreciate anything extra you think I could try there.